src/Analysis/Codelyzer.Analysis.CSharp/Handlers/MemberAccessExpressionHandler.cs

```diff
@@ -0,0 +1,25 @@
+using Codelyzer.Analysis.Common;
+using Codelyzer.Analysis.Model;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Linq;
+
+namespace Codelyzer.Analysis.CSharp.Handlers
+{
+    public class MemberAccessExpressionHandler : UstNodeHandler
+    {
+        private MemberAccess Model { get => (MemberAccess)UstNode; }
+
+        public MemberAccessExpressionHandler(CodeContext context,
+            MemberAccessExpressionSyntax syntaxNode)
+            : base(context, syntaxNode, new MemberAccess())
+        {
+            Model.Identifier = syntaxNode.ToString();
+            Model.Name = syntaxNode.Name?.ToString();
+
+            if (syntaxNode.Expression != null)
+            {
+                Model.Expression = syntaxNode.Expression?.ToString();
+            }
```

Comment:
nit: remove null conditional from `syntaxNode.Expression?.ToString();` as it is redundant or replace whole `if` block with `Model.Expression = syntaxNode.Expression?.ToString();` as it would be equivalent